files/default/dashboards/hops/user_statement_summaries.json

```diff
@@ -0,0 +1,203 @@
+      "spaceLength": 10,
+      "stack": true,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "rate(mysql_sys_user_total_statements{user=\"$user\", statement=~\"$statement\"}[1m])",
+          "format": "time_series",
+          "intervalFactor": 1,
+          "legendFormat": "{{ user }} - {{ statement }}",
+          "refId": "A"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "User statements",
+      "tooltip": {
+        "shared": true,
+        "sort": 0,
+        "value_type": "individual"
+      },
+      "type": "graph",
+      "xaxis": {
+        "buckets":null,
+        "mode":"time",
+        "name":null,
+        "show":true,
+        "values":[
+
+        ]
+      },
+      "yaxes": [
+        {
+          "format":"ops",
+          "label":null,
+          "logBase":1,
+          "max":null,
+          "min":null,
+          "show":true
+        },
+        {
+          "format":"short",
+          "label":null,
+          "logBase":1,
+          "max":null,
+          "min":null,
+          "show":true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    }
+  ],
+  "refresh": "5s",
+  "schemaVersion": 18,
+  "style": "dark",
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "allValue":"All",
+        "current":{
+
+        },
+        "datasource":"Prometheus",
+        "hide":0,
+        "includeAll":false,
+        "label":"User",
+        "multi":false,
+        "name":"user",
+        "options":[
+
+        ],
+        "query":"label_values(mysql_sys_user_total_statements{}, user)",
+        "refresh":1,
+        "regex":"",
+        "sort":1,
+        "tagValuesQuery":"",
+        "tags":[
+
+        ],
+        "tagsQuery":"",
+        "type":"query",
+        "useTags":false
+      },
+      {
+        "allValue":".*",
+        "current":{
+
+        },
+        "datasource":"Prometheus",
+        "hide":0,
+        "includeAll":true,
+        "label":"Statement/Operation",
+        "multi":true,
+        "name":"statement",
+        "options":[
+
+        ],
+        "query":"label_values(mysql_sys_user_total_statements{}, statement)",
+        "refresh":1,
+        "regex":"",
+        "sort":1,
+        "tagValuesQuery":"",
+        "tags":[
+
+        ],
+        "tagsQuery":"",
+        "type":"query",
+        "useTags":false
+      }
+    ]
+  },
+  "time": {
+    "from": "now-1h",
+    "to": "now"
+  },
+  "timepicker": {
+    "refresh_intervals": [
+      "5s",
+      "10s",
+      "30s",
+      "1m",
+      "5m",
+      "15m",
+      "30m",
+      "1h",
+      "2h",
+      "1d"
+    ],
+    "time_options": [
+      "5m",
+      "15m",
+      "1h",
+      "6h",
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
+  },
+  "timezone": "",
+  "title": "User Statement Summaries",
```

Comment:
I would call this dashboard "Database user statement summaries" and put it in the RonDB directory instead of the Hops directory. So that it's clear that the dashboard represents RonDB user statements and not other databases we have in Hopsworks.